src/util.cpp

```diff
@@ -936,3 +927,10 @@ int64_t GetStartupTime()
 {
     return nStartupTime;
 }
+
+fs::path GetAbsolutePathForArg(const fs::path& path, bool net_specific)
+{
+    if (!path.is_complete())
+        return GetDataDir(net_specific) / path;
```

Comment:
Would be a little better to return: `return fs::absolute(path, GetDataDir(net_specific))` to make it clear this returns an absolute path. You could also drop the `is_complete` check above if you did this.

Reply:
Great, thanks.